lib/util.js

```diff
@@ -67,6 +73,11 @@ module.exports.kickoff = function kickoff() {
           console.log(results);
           /* eslint-enable */
         }
+        done();
       }.bind(this));
+    }.bind(this);
   }.bind(this));
+
+  maxConcurrent = (baseProfile && baseProfile.maxConcurrent) || Infinity;
```

Comment:
Prior to this PR, `maxConcurrent` is implicitly `Infinity`. I think this is good as-is.